hydromt/flw.py

```diff
@@ -18,7 +18,7 @@
 logger = logging.getLogger(__name__)
 
 
-def flwdir_from_da(da, ftype="infer", check_ftype=True, mask=True):
+def flwdir_from_da(da, ftype="infer", check_ftype=True, mask=None):
```

Comment:
This may have impacts on the plugins as well no? If yes might be good to add to the checklist of #1

Reply:
Nevermind, I see you did a backwards compatibility as well